examples/minimal_client_service/src/minimal_client_async.rs

```diff
@@ -0,0 +1,34 @@
+use anyhow::{Error, Result};
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    let context = rclrs::Context::new(env::args()).unwrap();
```

Comment:
Could this be re-written to use `?` instead? I thought we moved away from using `unwrap` in the tutorials.

Reply:
Done in d2c72e3ab65e99fbe230ca4270003b1e4de3e225